<launch>
    <arg name="uav_name" default="MAV2" />

    <group ns="$(arg uav_name)">
        <param name="UAV_ID" value="2"/>
        <param name="cbf_gamma" value="1.0"/>
    	<param name="track_D" value="0.6"/>
        <param name="safe_D" value="0.5"/>
    	<param name="sub_topic" value="/vrpn_client_node/MAV2/pose"/>

        <include file="$(find mavros)/launch/px4.launch">
        </include>
        <node name="topic_tool" pkg="flight_control" type="topic_tool"></node>
        <node name="leader_track_cbf" pkg="flight_control" type="leader_track_cbf" output="screen">
            <rosparam subst_value="true">
                use_input: "velocity"  #velocity,position
           </rosparam>
       </node>
    </group>
</launch>
